### PR TITLE
📝  How to run docs

### LINKED ISSUE #635: Add “how to run” section in the README
…with focus on getting the storybook, react components and tests running.

### PR DESCRIPTION
**Pull request by wenche**
Resolves #635

### CODE REVIEW

**Comment by wenche** [comment]
Use ssh or https for cloning?

**Comment by vnys** [comment]
ssh

**Comment by vnys** [comment]
https://github.com/wenche/design-system/pull/3

**Comment by wenche** [comment]
I think this is sufficient for now at least for the React component part?

**Comment by vnys** [comment]
Yes, I think so too

**Review by vnys — APPROVED** [review]
A bit sneaky 😁

File: README.md [hunk 1]
```diff
@@ -31,7 +44,40 @@ While the design system itself lives in [Figma][], this repository contains impl
...
 | [Storefront](https://github.com/equinor/design-system/tree/develop/apps/storefront) (prod) | [![Build Status](https://dev.azure.com/equinor-design-system/Equinor%20Design%20System/_apis/build/status/storefront-production?branchName=develop)](https://dev.azure.com/equinor-design-system/Equinor%20Design%20System/_build/latest?definitionId=8&branchName=develop) | [![Version](https://img.shields.io/badge/dynamic/json?url=https://raw.githubusercontent.com/equinor/design-system/develop/apps/storefront/package.json&label=version&query=version&color=orange&prefix=v)](https://github.com/equinor/design-system/apps/storefront) |
 
+## How to run
 
+Would you like to contribute? Awesome! 👏
+
+We use a fork and pull-request workflow, so start with forking the repository on Github, then clone your fork.
+
+    $ git clone git@github.com:your-github-username/design-system
```
Comment (mimarz) [inline README.md:53]: Maybe set this up as a step 1 and 2? Something along  the line of

1. Fork https://github.com/equinor/design-system using the fork button
2. Clone locally using `git clone git@github.com:<your-github-username>/design-system`

**Review by mimarz — APPROVED** [review]
LGTM 👍, maybe just some rewording to clarify one step. Else it looks good

**Pull request merged**